trafilatura/main_extractor.py

```diff
@@ -12,6 +12,8 @@
 from lxml.etree import _Element, Element, SubElement, strip_elements, strip_tags, tostring
 from lxml.html import HtmlElement
 
+from urllib.parse import urljoin
```

Comment:
This is a standard package, ideally it should be imported before external packages like lxml. See the `isort` tool if you want to do this automatically.

Reply:
ok, fixed